apps/bandwagon/tests/test_models.py

```diff
@@ -138,6 +138,17 @@ def test_publishable_by(self):
         CollectionUser(collection=c, user=self.user).save()
         eq_(c.publishable_by(self.user), True)
 
+    def test_manager_publishable_by(self):
+        c1 = Collection.objects.create(author=self.user, name='B')
+        c2 = Collection.objects.create(author=self.user, name='A')
+        c3 = Collection.objects.create(author=self.other, name='D')
+        Collection.objects.create(author=self.other, name='C')
+        CollectionUser(collection=c1, user=self.user).save()
+        CollectionUser(collection=c2, user=self.other).save()
+        CollectionUser(collection=c3, user=self.user).save()
+        collections = Collection.objects.publishable_by(self.user)
+        eq_([c.name for c in collections], [c2.name, c1.name, c3.name])
```

Comment:
We use [pytest](https://pytest.org) for tests now, so even if most of the tests are still written using `eq_`, we rather use simple `assert`s now. Also, I think we can compare collections themselves, no need to compare their names:
`assert list(collections) == [c2, c1, c3]`